app/com/arpnetworking/rollups/RollupDefinition.java

```diff
@@ -164,13 +176,24 @@ public Builder setStartTime(final Instant value) {
         }
 
         /**
-         * Sets the {@code _groupByTags} and returns a reference to this Builder so that the methods can be chained together.
+         * Sets the {@code _allMetricTags} and returns a reference to this Builder so that the methods can be chained together.
+         *
+         * @param value the {@code _allMetricTags} to set
```

Comment:
Should these docstrings say `_filterTags` instead of `_allMetricTags`?

Reply:
Quite right, thanks!